fix(35/*) fix all navigation 404s (#1270)
Kong/docs.konghq.com: app/enterprise/0.35-x/plugins/oidc-auth0.md

## Auth0 IDP Configuration

This configuration will use a [client credentials grant][client-credentials-grant] as it is non-interactive, and because we expect clients to authenticate on behalf of themselves, not an end-user. To do so, you will need to [create an Auth0 API][create-auth0-api] and a [non-interactive client][non-interactive-client].

### API Configuration

When creating your API, you will need to specify an Identifier. Using the URL that your consumer makes requests to is generally appropriate, so this will typically be the hostname/path combination configured as an API in Kong.

After creating your API, you will also need to add the `openid` scope at `https://manage.auth0.com/#/apis/<API ID>/scopes`.

### Client Configuration

You will need to authorize your client to access your API. Auth0 will prompt you to do so after client creation if you select the API you created previously from the client's Quick Start menu. After toggling the client to Authorized, expand its authorization settings and enable the `openid` scope.

## Kong Configuration

> If you have not done so already, [create an API][create-api] to protect. The `url` configuration should match the Identifier you used when configuring Auth0.

Author: ```suggestion
If you have not done so already, [create a **Service**][add-service] to protect. The `url` configuration should match the Identifier you used when configuring Auth0.
```

  --data config.issuer="https://<auth0 API name>.auth0.com/.well-known/openid-configuration" \
  --data config.token_post_args_names="audience" \
  --data config.token_post_args_values="https://example.com/"
```

## Downstream configuration

The service accessing your resource will need to pass its credentials to Kong. It can do so via HTTP basic authentication, query string arguments, or parameters in the request body. Basic authentication is generally preferable, as credentials in a query string will be present in Kong's access logs (and possibly in other infrastructure's access logs, if you have HTTP-aware infrastructure in front of Kong) and credentials in request bodies are limited to request methods that expect
client payloads.

For basic authentication, use your client ID as the username and your client secret as the password. For the other methods, pass them as parameters named `client_id` and `client_secret`, respectively.

[client-credentials-grant]: https://auth0.com/docs/api-auth/tutorials/client-credentials
[create-auth0-api]: https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0
[non-interactive-client]: https://auth0.com/docs/clients

> [create-api]: /0.13.x/admin-api/#add-api

Author: ```suggestion
[add-service]: /enterprise/{{page.kong_version}}/getting-started/add-service
```
